src/chapters/introduction/best-practices.rst

```diff
@@ -153,20 +157,16 @@ Attention sports fans! Embrace your favorite quote(s):
 
 .. tip::
 
-   Your heros worked really hard, so should you. *Do your homework*.
+   Your heros worked really hard, so should you.
 
 Social Media
 -------------
 
-wnt motivation n 140 caraters r less? Try thez (LOL):
+Want motivation in 140 characters or less? Check out these posts:
```

Comment:
The shorter sentence (wnt motivation n 140 caraters r less? Try thez (LOL)) came from suggestions from my daughter, comments from the LCHS teachers, and a desire for some irony.

Also, to remain consistent with the other sections, we should keep the pull quote, but we can change the comment to something else.  (I think it should be abbr, tho).

Reply:
I added a tip! I changed the tips on the page so that they didn't all say "Do the homework" because when I read the page, it felt like I was being yelled at by the book to do my homework. Which if the book stood alone, I would get. But the book is going to be used in a classroom where the teachers will be there to tell them to do the homework. The additions to the tips I found quite helpful and a good way to highlight a takeaway that I can see students writing down or committing to memory. Hopefully, the social media tip serves that purpose as well!

In regards to the joke, I did get it, however, I remember working with high schoolers and seeing similar jokes in many textbooks. In my experience, those jokes only had two reactions. Either a student laughed and moved on or a student rolled their eyes and decided the whole book wasn't worth their time. I am not sure it is worth it to make the first student laugh if it is at the risk of the second student disengaging.